package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@exodus/secp256k1",
-  "version": "5.0.0-exodus.0",
+  "version": "5.0.0-exodus.3",
```

Comment:
i'm curious why we keep bumping this. Are we doing prereleases from this PR? Let's make that clear, e.g. with 5.0.0-exodus-rc.3 and not commit it in this PR

Reply:
Okay didn't know about that